pages/Hazmat-Suit.md

```diff
@@ -0,0 +1,16 @@
+The Hazmat suit is part of the [Armor](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor) category.<br>
+It consists of four parts (listed below).<br><br>
+
+| Piece  |  Bonus effects  |
+| ------------- | -------------  |
+| Scuba Helmet  |  Water breathing  |
+| Hazmat Suit  |  Fire Resistance  |
+| Hazmat Suit Leggings |  N/A  |
+| Rubber Boots |  N/A  |
+
+## Obtaining
+Each piece can be crafted in the [Armor Forge](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor-Forge).
+
+## Usage
+This set is mostly used to provide protection against [Radiation](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Radiation).<br>
```

Comment:
You should probably add that you need to wear the full set in order for this to happen :?

Reply:
Yup I'll mention that